From 40a9ae30fc0fa2c1cfb2c2a1cfc797336c9acff0 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Tue, 8 Nov 2016 09:25:51 +0800
Subject: [PATCH 188/188] net: macb: Correct CAPS masks

The commit d470dfb3 comes from:
  https://github.com/Xilinx/linux-xlnx.git

USRIO and JUMBO have the same capability mask.
Correct this and re-arrange the definitions to be in order.

Signed-off-by: Harini Katakam <harinik@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Zumeng Chen  <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/cadence/macb.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/cadence/macb.h b/drivers/net/ethernet/cadence/macb.h
index e8f6781..f300e15 100644
--- a/drivers/net/ethernet/cadence/macb.h
+++ b/drivers/net/ethernet/cadence/macb.h
@@ -463,13 +463,13 @@
 #define MACB_CAPS_NO_GIGABIT_HALF		0x00000008
 #define MACB_CAPS_USRIO_DISABLED		0x00000010
 #define MACB_CAPS_JUMBO				0x00000020
+#define MACB_CAPS_TSU				0x00000030
+#define MACB_CAPS_PCS				0x00000040
 #define MACB_CAPS_FIFO_MODE			0x10000000
 #define MACB_CAPS_GIGABIT_MODE_AVAILABLE	0x20000000
 #define MACB_CAPS_SG_DISABLED			0x40000000
 #define MACB_CAPS_MACB_IS_GEM			0x80000000
 
-#define MACB_CAPS_TSU				0x00000020
-#define MACB_CAPS_PCS				0x00000040
 #define NS_PER_SEC				1000000000ULL
 
 /* Bit manipulation macros */
-- 
2.0.2

